test/expected/templates/context.html

```diff
@@ -0,0 +1,3 @@
+<div>root</div>
+<div>variable</div>
+
```

Comment:
This is just dumb, but no idea of how to get rid of this empty line without tests failures